plugins/usbdmx/EuroliteProFactory.cpp

```diff
@@ -88,13 +112,22 @@ bool EuroliteProFactory::DeviceAdded(
   // Eurolite USB-DMX512-PRO MK2?
   } else if (descriptor.idVendor == VENDOR_ID_MK2 &&
              descriptor.idProduct == PRODUCT_ID_MK2) {
-    if (m_enable_eurolite_mk2) {
-      OLA_INFO << "Found a possible new Eurolite USB-DMX512-PRO MK2 device";
-      LibUsbAdaptor::DeviceInformation info;
-      if (!m_adaptor->GetDeviceInfo(usb_device, descriptor, &info)) {
-        return false;
-      }
+    LibUsbAdaptor::DeviceInformation info;
+    if (!m_adaptor->GetDeviceInfo(usb_device, descriptor, &info)) {
+      return false;
+    }
+
+    const bool serial_matches =
+      STLContains(m_expected_eurolite_mk2_serials, info.serial);
 
+    if (m_enable_eurolite_mk2 || serial_matches) {
+      if (serial_matches) {
+        OLA_INFO << "Enabling new Eurolite USB-DMX512-PRO MK2 device "
```

Comment:
We should probably be slightly pessimistic here, as the manufacturer and product name checks could still fail, e.g. "Found a probable..."